src/style.css

```
@@ -0,0 +1,148 @@
+
+@font-face {
+  font-family: Roboto;
+  src: url("./font/Roboto-Medium.ttf") format("truetype");
+  font-weight: 500;
+  font-style: medium;
+}
+
+@font-face {
+  font-family: Roboto-Normal;
+  src: url("./font/Roboto-Regular.ttf") format("truetype");
+  font-weight: 400;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Roboto-Bold;
+  src: url("./font/Roboto-Bold.ttf") format("truetype");
+  font-weight: 700;
+  font-style: bold;
+}
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
```

```suggestion
* {
  box-sizing: border-box;
}
```
in the `*` we can use only `box-sizing`